datalad_tabby/resources/contexts/common.jsonld

```diff
@@ -10,5 +10,23 @@
   "last-updated": "schema:dateModified",
   "license": "schema:license",
   "name": "schema:name",
+  "url": "schema:url",
   "version": "schema:version",
+  "address": "",
+  "affiliation": "",
+  "annex-key": "",
+  "author": "",
+  "citation": "",
+  "checksum[md5]": "",
```

Comment:
I want to include the term from https://ontobee.org/ontology/NCIT?iri=http://purl.obolibrary.org/obo/NCIT_C171276

Here's more info about the ontology: https://ontobee.org/ontology/NCIT

However, I am unsure how to add the the key-value pair for the specific vocabulary in this context. Would it be something like: `"obo": "http://purl.obolibrary.org/obo/"`, and then the term is `"checksum[md5]": "obo:NCIT_C171276"`?

Reply:
yes, looks good